pkg/management/postgres/instance.go

```diff
@@ -447,10 +449,22 @@ func (instance Instance) WithActiveInstance(inner func() error) error {
 	if err != nil {
 		return fmt.Errorf("while activating instance: %w", err)
 	}
+
+	ctx, ctxCancel := context.WithCancel(context.Background())
+	csvPipe := logpipe.NewLogPipe()
+
+	go func() {
+		if err := csvPipe.Start(ctx); err != nil {
```

Comment:
this will always fail when WithActiveInstance is called from CompleteCrashRecovery, but that's not a real issue.

Reply:
Actually this could be an issue as log pipes don't know whether they are already running or not, so multiple goroutines could be spawned and attached to the same file. Given the current usages of WithActiveInstance this should never happen afaict, but it could happen in the future if the code is changed.